src/components/CalendarComponents/categoriesBar.vue

```diff
@@ -25,24 +25,14 @@
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator';
+import { Component, Vue, Prop } from 'vue-property-decorator';
+
 import Category from '@/models/Category';
-import { AllCategories } from '@/db-service/Categories/queries';
 
-@Component({
-  apollo: {
-    allCategories: {
-      query: AllCategories,
-    },
-  },
-})
+@Component
 export default class CategoriesBar extends Vue {
-  get categories(): Category[] {
-    if (this.$data.allCategories !== undefined) {
-      return this.$data.allCategories.nodes;
-    }
-    return [];
-  }
+    @Prop()
+    categories!: Array<Category>
```

Comment:
There is a reason to use props? Categories should be in the categories bar component

Reply:
It has something to do a pervious try and I accidently kept it that way, but as I saw on the spinner pull request the query will be that way so I won't change it